### DISCUSSION TITLE
⛔ Feature Request: Enable/Disable fix of export leads to csv

### DISCUSSION THREAD

**Discussion by Mulli** [elementor/elementor#21322] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The fix introduced in E-PRO 3.11 is welcome and important.
However, **the fix breaks backword compatiblity**.
Which, in turn, diable receiveing additional version updates.

In my case, a lot of software depends on the old (bad) order of columns (see issue #17775 for more details).

### Describe the solution you'd like

It would be nice if:
In experimets - an option to enable/disable the fix will allow to receive updates for E-Pro.
Currently - it must be freezed on Version 3.10.3
Until software update (expensive) is performed in client software.

### Describe alternatives you've considered

In v 3.10.3: /plugins/elementor-pro/modules/forms/submissions/export/csv-exports.php on lines 109 and 145 you have
ksort( $row );
By commenting this out this piece of code elementor stops sorting the values and it will maintain the order from the database :)

So you can add that to achieve backword compatibility - but this is code change. grrrrrrrrrr....

**Comment by nicholaszein** [comment]
Hello @Mulli,

Thank you for submitting a feature request! 🙏 

### ⛔ 🚀 After reviewing your request, we have decided not to add this to our roadmap for the time being.

Please feel free to contact us again with other feature suggestions. 😉 

Best regards
